charts/stable/makemkv/docs/installation_notes.md

```diff
@@ -8,3 +8,6 @@
 ---
 
 - If you are passing through devices such as `Optical Drives`, you have to Click `Container Security Settings` and set `PUID` to `0`.
+- Mount USB Devices
+  Host Device Path `/dev/dvd`
+  Container Device Path `/dev/dvd`
```

Comment:
```suggestion
    - Host Device Path `/dev/dvd` (Adjust this based on when your device is mounted on the host)
    - Container Device Path `/dev/dvd` (This must be exactly like that)
```

Reply:
I did some more testing and this seems to work also
  Host Device Path `/dev/sg0`
  Container Device Path `/dev/sg0`
I have not tried mixing and matching